sway/commands/input/rotation_angle.c

```diff
@@ -0,0 +1,29 @@
+#include <math.h>
+#include <stdlib.h>
+#include <string.h>
+#include "sway/config.h"
+#include "sway/commands.h"
+#include "sway/input/input-manager.h"
+#include "util.h"
+
+struct cmd_results *input_cmd_rotation_angle(int argc, char **argv) {
+	struct cmd_results *error = NULL;
+	if ((error = checkarg(argc, "rotation_angle", EXPECTED_AT_LEAST, 1))) {
+		return error;
+	}
+	struct input_config *ic = config->handler_context.input_config;
+	if (!ic) {
+		return cmd_results_new(CMD_FAILURE, "No input device defined.");
+	}
+
+	float rotation_angle = parse_float(argv[0]);
+	if (isnan(rotation_angle)) {
+		return cmd_results_new(CMD_INVALID,
+			"Invalid rotation_angle; expected float.");
+	} if (rotation_angle < 0 || rotation_angle > 360) {
+		return cmd_results_new(CMD_INVALID, "Input out of range [0, 360]");
```

Comment:
Since the valid range for `rotation_angle` is between 0.0 (inclusive) and 360.0 (exclusive), should the error string be:
`"Input out of range [0, 360)"`

Reply:
I'm not sure what you mean, that is the error message that we are retuning, no? I'm confused. Or do you want me to change?
```suggestion
		return cmd_results_new(CMD_INVALID, "Input out of range [0, 360)");
```

Reply:
Sorry if my comment was ambiguous, it was mostly a nit. Seems you understood my comment though. It was just ensuring that the upper bound is non-inclusive in the error message (i.e. `]` -> `)`)

Reply:
Okay, just want to be sure :) added your suggestion.